protocol/x/sending/keeper/msg_server_create_transfer_test.go

```diff
@@ -256,3 +256,69 @@ func TestWithdrawFromSubaccount(t *testing.T) {
 		})
 	}
 }
+
+func TestMsgServerSendFromModuleToAccount(t *testing.T) {
+	tests := map[string]struct {
+		// Setup.
+		testMsg    types.MsgSendFromModuleToAccount
+		keeperResp error // mock keeper response
+		// Expectations.
+		expectedResp *types.MsgSendFromModuleToAccountResponse
+		expectedErr  string
+	}{
+		"Success": {
+			testMsg: types.MsgSendFromModuleToAccount{
+				Authority:        constants.GovModuleAccAddressString,
+				SenderModuleName: "community_treasury",
+				Recipient:        constants.AliceAccAddress.String(),
+				Coin:             sdk.NewCoin("dv4tnt", sdk.NewInt(1)),
+			},
+			expectedResp: &types.MsgSendFromModuleToAccountResponse{},
+		},
+		"Failure: invalid authority": {
+			testMsg: types.MsgSendFromModuleToAccount{
+				Authority:        "12345",
+				SenderModuleName: "community_treasury",
+				Recipient:        constants.AliceAccAddress.String(),
+				Coin:             sdk.NewCoin("dv4tnt", sdk.NewInt(1)),
+			},
+			expectedErr: fmt.Sprintf(
+				"invalid authority %s",
+				"12345",
+			),
+		},
+		"Failure: keeper method returns error": {
+			testMsg: types.MsgSendFromModuleToAccount{
+				Authority:        constants.GovModuleAccAddressString,
+				SenderModuleName: "community_treasury",
+				Recipient:        constants.CarlAccAddress.String(),
+				Coin:             sdk.NewCoin("dv4tnt", sdk.NewInt(1)),
+			},
+			keeperResp:  fmt.Errorf("keeper error"),
+			expectedErr: "keeper error",
+		},
+	}
+
+	for name, tc := range tests {
```

Comment:
Let's prefer to not have mocked unit tests unless it's really hard to set up. Setting balances with a real keeper and checking balance afterwards should be relatively simple. 

Consider using the `testapp` framework which enables setting up account balances through genesis state ([example](https://github.com/dydxprotocol/v4-chain/blob/60d09718d807189f9ebc8522ac9cb15e66c6d8e9/protocol/x/rewards/keeper/keeper_test.go#L600-L616))